assets/js/main.js

```diff
@@ -1,29 +1,38 @@
-
 (function () {
-
 	// add more files here if needed
-        var files = ['javascript.json', 'text-editors.json', 'jquery.json', 'node.json']
+    var files = ['javascript.json', 'node.json', 'jquery.json', 'text-editors.json']
+
+    files.forEach(function (filename) {
+        // create placeholder divs to enforce topic order
+        var el = getTargetElement(filename.split('.')[0])
+        document.getElementById('content').appendChild(el)
 
-	.forEach(function (filename) {
-		getFile(filename).then(function (data) {
-			render(data)
-		})
-	})
+        getFile(filename).then(function(data) {
+            render(data);
+        })
+    })
 
 	function render (data) {
 		var template = Handlebars.compile(document.getElementById('main_template').text)
 		var html = template(data)
-		var el = document.createElement('div')
+
+        var id = 'topic-' + data.title.replace(/ /g, '-')
+		var el = document.getElementById(id)
 		el.innerHTML = html
-		document.getElementById('content').appendChild(el)
 	}
 
-	function getFile (filename) {
-		return reqwest({
-			url: '/data/' + filename,
-			type: 'json',
-			method: 'get'
-		})
-	}
+    function getFile (filename) {
+        return reqwest({
+            url: '/data/' + filename,
+            type: 'json',
+            method: 'get'
+        })
+    }
+
+    function getTargetElement(id) {
```

Comment:
Keep the code style consistent with line 24. Do you want a space before `(` or not.